code/modules/antagonists/changeling/changeling.dm

```diff
@@ -364,7 +364,6 @@
 /datum/antagonist/changeling/greet()
 	if (you_are_greet)
 		to_chat(owner.current, "<span class='boldannounce'>You are [changelingID], a changeling! You have absorbed and taken the form of a human.</span>")
-	to_chat(owner.current, "<span class='boldannounce'>Use say \"[MODE_TOKEN_CHANGELING] message\" to communicate with your fellow changelings.</span>")
```

Comment:
MODE_TOKEN_CHANGELING could probably be removed too

Reply:
done